scripts/change_url

```diff
@@ -18,6 +18,16 @@ ynh_script_progression --message="Updating NGINX web server configuration..." --
 
 ynh_change_url_nginx_config
 
+#=================================================
+# SPECIFIC MODIFICATIONS
+#=================================================
+# RECONFIGURING MATOMO
+#=================================================
+ynh_script_progression --message="Reconfiguring Matomo..." --weight=2
+
+# See https://matomo.org/faq/how-to-install/faq_18/
+ynh_replace_string --match_string="$old_domain" --replace_string="$new_domain" --target_file="$install_dir/config/config.ini.php"
```

Comment:
```suggestion
if [[ ! -f "$install_dir/config/config.ini.php" ]]; then
	# config.ini.php is only created during the post-install process...
	# it is therefore not present when the CI tests are carried out...
	# This condition is only for CI test to go through the upgrade process
	ynh_replace_string --match_string="$old_domain" --replace_string="$new_domain" --target_file="$install_dir/config/config.ini.php"
fi
```

Reply:
I think this will fail again because the condition checks that the file does NOT exist and attempts to do the replace if the file doesn't exist.